### DISCUSSION TITLE
Convert 8bit image with [0,1]value to [0, 2]value image

### DISCUSSION THREAD

**Discussion by bemoregt** [ImageMagick/ImageMagick#2159] [Help] [answered]
Hi, @dlemstra @alex @mirakui @yoya @rudchenkos 

How can I convert 8bit image with [0,1] value to [0, 2] valued image?
or
How can I convert 8bit image with [0,255] value to [0, 2] valued image?

using imagemagick.

Thanks in advance.
Best,
@bemoregt.

**Comment by snibgo** [comment, marked as answer]
What version of IM? What is the text output of "magick -version" or "convert -version". On what platform?

Assuming you have Q8:

-evaluate Multiply 2

-evaluate Divide 127.5

If you have a different Q, eg Q16, the answers are the same but may not be what you actually want.

**Reply by bemoregt** [reply]
Hi @snibgo 

My Machine is OS Mojave
Version: ImageMagick 7.0.10-0 Q16 x86_64 2020-03-08 https://imagemagick.org
Copyright: © 1999-2020 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(3.1) 
Delegates (built-in): bzlib freetype heic jng jp2 jpeg lcms ltdl lzma openexr png tiff webp xml zlib

My Goal is making mask for Mask R-CNN datasets.

Thanks.
Best,
@bemoregt.
